Makefile

```diff
@@ -15,12 +15,16 @@ export TARGET
 help: ## Display this help text
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-install: node_modules ## Install dependencies locally
+install: node_modules .scripts ## Install dependencies locally
 
 node_modules: package.json package-lock.json
 	npm install
 	touch node_modules
 
+.scripts: .gitmodules
+	git submodule update --init --recursive .scripts
+	touch .scripts
```

Comment:
what's this for?

Reply:
(sounds like `make` keeping track of modified files/directories for caching purposes)

Reply:
`.gitmodules` is only likely to change when there are new submodules (if ever) though.

Reply:
`.scripts` has a dependency on the `.gitmodules` file, `make .scripts` will only run if the modification time on `.gitmodules` is newer than `.scripts`.

Reply:
> `.gitmodules` is only likely to change when there are new submodules (if ever) though.

Hmm, good point. Let's test it...

Reply:
You're right, it won't change. Will have to not include this bit otherwise future changes won't work.

Reply:
https://github.com/libero/article-store/pull/34/commits/9b2cfdb713142dcf8fd7743b4820a828234f03df

Reply:
Correct thing would be to rely on `.git/modules/.scripts` (or something inside), but doesn't seem worth it.